test/dashboard-e2e/helpers/common-helpers.ts

```diff
@@ -15,4 +15,16 @@ export class CommonHelpers {
             }
         }
     }
+
+    static validateTestSuite(testSuiteData, createdTestSuiteData) {
+        expect(testSuiteData.name).toEqual(testSuiteData.name);
+        expect(testSuiteData.description).toEqual(createdTestSuiteData.description);
+        CommonHelpers.validateLabels(testSuiteData.labels, createdTestSuiteData.labels);
+    }
+
+    static validateLabels(labels, createdLabels) {
+        for (const [name, value] of Object.entries(labels)) {
```

Comment:
You need to check as well if the length is the same

Reply:
I will add it